core/src/main/java/com/netflix/conductor/core/execution/AsyncSystemTaskExecutor.java

```diff
@@ -74,6 +74,12 @@ public void execute(WorkflowSystemTask systemTask, String taskId) {
             return;
         }
 
+        LOGGER.info(
```

Comment:
When will this log print ? Galaxy is already throwing lot of logs, adding more info logs is not viable. Can you please comment why this log is needed as info.

Reply:
remove this log